frontend/public/components/masthead-toolbar.jsx

```diff
@@ -238,13 +265,21 @@ class MastheadToolbar_ extends React.Component {
 
   render() {
     const { isApplicationLauncherDropdownOpen, isHelpDropdownOpen, showAboutModal } = this.state;
-    const { flags } = this.props;
+    const { flags, activeNamespace } = this.props;
     const resources = [{
       kind: clusterVersionReference,
       name: 'version',
       isList: false,
       prop: 'obj',
     }];
+
+    let serviceMeshItem;
+    if (activeNamespace && activeNamespace !== ALL_NAMESPACES_KEY) {
+      serviceMeshItem = <Firehose key="firehose" resources={[{kind: 'Namespace', name: activeNamespace, prop: 'namespace', isList: false}]}>
```

Comment:
We shouldn't need firehose. We already have the namespace name.

Reply:
yes, but I need the namespace Annotations

Reply:
OK we might want to put the current selected project or namespace in redux instead of opening a new watch for it. We already have the resource in the namespace dropdown.

Reply:
Ok, is there some example code I could look at which does this?  -thanks

Reply:
The namespace name is set in redux here:

https://github.com/openshift/console/blob/master/frontend/public/components/namespace.jsx#L283

We might change the existing `setActiveNamespace` action and reducer to keep the namespace obj, too. (But do it in a way that doesn't require everything using `getActiveNamespace` to change.)